modules/database/main.tf

```diff
@@ -33,8 +33,15 @@ resource "aws_db_instance" "pg_db" {
   deletion_protection          = var.deletion_protection
   performance_insights_enabled = true
   storage_encrypted            = true
+  backup_retention_period      = var.db_retention_period
+
+
+  restore_to_point_in_time {
+    restore_time                  = var.apply_pitr_backup_rds ? var.pitr_restore_time : null
+    source_db_instance_identifier = var.apply_pitr_backup_rds ? aws_db_parameter_group.postgres_db.id : null
```

Comment:
Expose this as a parameter - `rds_restore_to_point_in_time_source_db_instance_identifier`